src/main/antlr4/Manifold.g4

```diff
@@ -89,14 +92,23 @@ lvalue:
   ;
 
 // TODO: declarations as expressions
-expression: /* declaration | */ rvalue;
+expression:
+  rvalue
+  // | declaration
+  ;
+
+statement:
+    expression #ExpressionStatment
+    | 'import' STRING_VALUE #ImportStatement
+    ;
+
+STATEMENT_TERMINATOR: ';';
```

Comment:
Statement was just the closest analog I had for naming this rule. They aren't executed sequentially or anything, but we need a grammar rule above expression. `import`s can't be expressions because those are allowed to be contained in tuples. Function definitions should also probably not be allowed inside of tuples too.